libraries/chain/include/graphene/chain/hardfork.hpp

```diff
@@ -21,3 +21,7 @@
 #ifndef UNSUCCESSFUL_CANDIDATE_DIFFPOINT
 #define UNSUCCESSFUL_CANDIDATE_DIFFPOINT 1765771
 #endif
+// AUTO GAS
+#ifndef UNAUTO_GAS
+#define UNAUTO_GAS (fc::time_point_sec( 1585826672 ))
+#endif 
```

Comment:
Why use such name as UNAUTO_GAS?  Would be much better to use a more explicit name.